### ISSUE TITLE
any chance for the modulators to work?

### ISSUE THREAD

**Issue by coderofsalvation** [felixroos/sfumato#1]
Thank you for making this.
Sending midi CC to soundfonts would be so awesome.

**Comment by felixroos** [comment]
yes that would be nice! i am not sure how far the underlying soundfont2 package supports modulators, but there is at least some [source code](https://mrtenz.github.io/soundfont2/api/modulator.html). I have not yet used soundfonts with modulators, do you have specific ones to test modulators with?

**Comment by coderofsalvation** [comment]
yes, i can make a simple sf2 with standard midi cc's (volume & pan are probably the first lowhanging fruits) along with a midifile.

I think the 'hardest' cc's are reverb & chorus send, but maybe a nice 'cheat' would be using the native webaudio nodes for that (would be a nice performance-win compared to fluidsynth)

**Comment by felixroos** [comment]
that would be nice to have something for testing, tbh i have not done much yet with soundfonts so i am learning how this works by implementing 🙂the main purpose of this lib is to be used by strudel, example https://strudel.tidalcycles.org/?u7qAdlwp3Qig using native webaudio nodes for the default behavior sounds good, the sampling engine is also implemented with buffer nodes
